test/routes/sessions_routes_test.rb

```diff
@@ -25,6 +25,10 @@ class SessionsRoutesTest < ActionController::TestCase
     assert_routing "/logout", { controller: "shopify_app/sessions", action: "destroy" }
   end
 
+  test "patch_shopify_id_token routes to sessions#patch_shopify_id_token" do
```

Comment:
Nit: this file is testing `session` routes (in the sense of log in/log out/`request.session`, not session token I guess 😅). Maybe we could create a new file?

Reply:
hmmm I think it belongs here since it's just testing the routes, and we did add the new route to patch session token 🤔

Reply:
I just mean there are many files testing routes like `test/routes/callback_routes_test.rb` and `test/routes/webhooks_routes_test.rb`, each testing different route groups. The `session_routes_test.rb` file is testing login/logout, so I think we _could_ have an `id_token_routes_test.rb`, wdyt?

Reply:
Oh sorry I missed it was in the same controller. I think they're different but not sure if it warrants its own controller, never mind this!